bin/doctrine-pear.php

```diff
@@ -1,5 +1,14 @@
 <?php
 
+fwrite(
+    STDERR,
+    '[Warning] The use of this script is discouraged.'
+    . ' You find instructions on how to boostrap the console runner in our documentation.'
```

Comment:
You will find?

Reply:
This is the same wording as we use in DBAL. I'm not a native speaker, but it sounds correct to me.

Reply:
I'd like to have the "You will find" correction here too. It sounds weird in the DBAL wording.

Reply:
@morozov WDYT? As someone who lives in the States?

Reply:
I'd put it as "See <url> for instructions on bootstrapping the console runner." (typo in "boostrap").

Reply:
Thanks, @morozov. I've updated the PR with your suggestion.